CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+Firstly, thank you for taking time and contributing to the project. All your efforts to contribute are highly appreciated!
+
+## Feature Requests
+
+Feature Requests by the community are highly encouraged. However, please make sure to check that your feature is not already on the [roadmap](https://github.com/angelinag/protecc/projects/1). Clearly state why this feature is necessary or why it will be a great addition to the project along with sample use cases to help developers better understand it.
+
+## Reporting an issue
+
+Before submitting an issue please make sure:
+
+- You have read the documentation of the project.
+- You have already searched for related [issues](https://github.com/angelinag/protecc/issues), and found none open (if you found a related closed issue, please link to it from your post).
+- Your issue title is concise, on-topic and polite.
+- You can and do provide steps to reproduce your issue.
+- For developers, please make sure your environment is setup correctly.
+
+## Workflow for submitting a PR
+
+1. Fork the repository to your own GitHub account
+
+2. Clone from your repository
+
+```bash
+    $ git clone https://github.com/your-username/repository-name
```

Comment:
change this to contain the repo url

Reply:
This URL will the forked copy right? So we won't know what the username would be.

Reply:
I'll update the repository name assuming that's not going to change 🤭